observations/README.md

```diff
@@ -0,0 +1,1279 @@
+                    "code": "1005671000000105",
+                    "display": "Serum cholesterol level"
+                }
+            ],
+            "text": "Serum cholesterol"
+        },
+        "subject": {
+            "reference": "Patient/80bd1375-a1e3-4137-95ff-fb3316cf3496"
+        },
+        "effectiveDateTime": "2001-03-30",
+        "issued": "2010-02-09T12:31:51.000+00:00",
+        "performer": [
+            {
+                "reference": "Practitioner/C5DEFBF3-0174-BC6F-182C-B777B9C6FF43"
+            }
+        ],
+        "valueQuantity": {
+            "value": 6.1,
+            "unit": "millimole per liter",
+            "code": "mmol/L"
+        },
+        "related": [
+            {
+                "type": "derived-from",
+                "target": {
+                    "reference": "Observation/31465FC9-15B2-4391-A5EF-8F70C2154AAF"
+                }
+            }
+        ]
+    }
+}
+```
+
+</details>
+
+# JSON FHIR > XML HL7
+
+FHIR Observations can be mapped to the following HL7 components. The mapping for each GP Connect 
+observation type (in **bold**) is described in the sections below:
+
+- ObservationStatement (**[Uncategorised Data](#Uncategorised-Data-(JSON-FHIR->-XML-HL7))** and **[Test Result](#Test-Result-(JSON-FHIR->-XML-HL7))**)
+- CompoundStatement (**[Blood Pressure](#Blood-Pressure-(JSON-FHIR->-XML-HL7))**, **[Test Group Header](#Test-Group-Header-(JSON-FHIR->-XML-HL7))**)
+- NarrativeStatement (**[Comment Note](#Comment-Note-(JSON-FHIR->-XML-HL7))** / **[Filing Comment](#Filing-Comment-(JSON-FHIR->-XML-HL7))**)
+
+## Uncategorised Data (JSON FHIR > XML HL7)
+
+Uncategorised data is mapped from an FHIR `Observation` to a HL7 `ObservationStatement` where it is not deemed to be a 
+Blood Pressure or Comment Note. 
+
+| Mapped to (XML HL7 ObservationStatement)             | Mapped from (JSON FHIR / other source )                                                                                                                                         |
+|------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| id \[@root]                                          | Unique ID generated by the adaptor                                                                                                                                              |
+| code                                                 | `Observation.code.coding` as described in the FHIR > XML section of [Codeable Concept](../codeable%20concept/README.md)                                                         |
+| statusCode                                           | fixed value = `"COMPLETE"`                                                                                                                                                      |
```

Comment:
I am not sure whether this status is set as I can't find any references in code?